opennlp-tools/src/test/java/opennlp/tools/eval/Conll00ChunkerEval.java

```diff
@@ -45,8 +47,11 @@
  * <a href="http://www.cnts.ua.ac.be/conll2000/chunking/"> site </a>
  * and decompress them into this directory: $OPENNLP_DATA_DIR/conll00.
  */
-public class Conll00ChunkerEval {
+public class Conll00ChunkerEval extends AbstractEvalTest {
 
+  private static final File TEST_DATA_FILE = new File(getOpennlpDataDir(), "conll00/test.txt");
```

Comment:
It is nicer if this fails not during class loading but while running verifyTrainingData